views/content-rendered-post.phtml

```diff
@@ -1,9 +1,19 @@
 <h1><?= $data->title ?></h1>
 <?= $data->content ?>
-<?php
-if ($data->cross_link) {
-	?>
+<?php if ( $data->cross_link ) : ?>
 	<hr>
-	<p><em>Originally published at <a href="<?= $data->permalink ?>"><?= $data->site_name ?></a>.</em></p>
-	<?php
-}
+	<p>
+		<em>
+			<?php
+			printf(
```

Comment:
formatted as two lines, parentheses spacing, double quotes (and elsewhere in this patch).